build-jenkins-agent-windows.pkr.hcl

```diff
@@ -90,6 +90,13 @@ build {
     pause_before = "1m"
   }
 
+  ## enable longpath
+  provisioner "powershell" {
+    elevated_user     = local.windows_winrm_user[var.image_type]
+    elevated_password = build.Password
+    inline = ["Set-ItemProperty -Path 'HKLM:\\SYSTEM\\CurrentControlSet\\Control\\FileSystem' -Name LongPathsEnabled -type DWord -Value 1"]
```

Comment:
This line breaks the packer syntax (see the errors in the `packer init` stage) because the `\\` is an escape character for a litteral slash in terraform (ref. https://developer.hashicorp.com/terraform/language/expressions/strings)